RELEASE.md

```diff
@@ -4,6 +4,10 @@ TensorFlow 2.1 will be the last TF release supporting Python 2. Python 2 support
 
 ## Major Features and Improvements
 * The `tensorflow` pip package now includes GPU support by default (same as `tensorflow-gpu`) for both Linux and Windows. This runs on machines with and without NVIDIA GPUs. `tensorflow-gpu` is still available, and CPU-only packages can be downloaded at `tensorflow-cpu` for users who are concerned about package size.
+* **Windows users:** officially-released `tensorflow` Pip packages are now built with Visual
+  Studio 2019 version 16.4 in order to take advantage of the new `/d2ReducedOptimizeHugeFunctions` compiler flag. To use these new packages, you must install "Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019", available from Microsoft's website [here](https://support.microsoft.com/help/2977003/the-latest-supported-visual-c-downloads).
+  * This does not change the minimum required version for building TensorFlow from source on Windows.
```

Comment:
Although it does mean that you cannot to EIGEN_STRONG_INLINE, right?

Reply:
Good point, `EIGEN_STRONG_INLINE` takes very long to compile without this new flag. I'll update the description for clarity.

Reply:
Ok, improved.